assets/section-main-product.css

```diff
@@ -383,6 +405,13 @@ a.product__text {
   }
 }
 
+@media screen and (min-width: 990px) {
+  .different-color-scheme {
+    padding: 3rem;
```

Comment:
What are your thoughts on using `4rem` which is also being applied to the `product__info-wrapper`? It just looks a little bit off to me but I'm curious to hear your thoughts:

https://github.com/Shopify/dawn/blob/73991f6a031a4443cf75b7f559e33f13e4af1dd6/assets/section-main-product.css#L65-L67

Here's a quick video for comparison:

https://user-images.githubusercontent.com/28404165/186766069-bac5d841-5635-4e61-8a9c-add85db8c33a.mp4